app/src/main/res/values/strings.xml

```diff
@@ -650,5 +649,11 @@
     <string name="cloud_plugin_google_play_uri" translatable="false">market://details?id=com.filemanager.amazecloud</string>
     <string name="cloud_plugin_google_play_web_uri" translatable="false">https://play.google.com/store/apps/details?id=com.filemanager.amazecloud</string>
     <string name="failed_install_apk">Failed to install</string>
+
+    <!-- from https://android.googlesource.com/platform/frameworks/base/+/master/core/res/res/values/strings.xml -->
+    <!-- Storage description for internal shared storage. [CHAR LIMIT=NONE] -->
+    <string name="storage_internal">Internal shared storage</string>
```

Comment:
Can we keep the original "Internal Storage" instead? This will look bad on low ppi devices.

Reply:
This change came from @EmmanuelMess, but I agree that "Internal Storage" looks better